src/main/resources/db/migration/V20221004093000__create_table_pay.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE pay
+(
+    cardNr         VARCHAR2(16) PRIMARY KEY,
+    salary         NUMBER(8, 2),
+    commission_pct NUMBER(2, 2)
+);
+
+COMMENT ON TABLE PAY IS 'Pay table which contains employees card number, salary, and commission';
+COMMENT ON COLUMN pay.cardNr IS 'Primary key of pay table';
+COMMENT ON COLUMN pay.salary IS 'Salary of employee';
+COMMENT ON COLUMN pay.commission_pct IS 'Commission of employee';
+
+ALTER TABLE EMPLOYEES
+    ADD
+        cardNr VARCHAR2(16)
+            CONSTRAINT EMP_CARD_FK REFERENCES PAY (cardNr) UNIQUE;
+
+UPDATE pay
```

Comment:
According to condition you need to transfer commission_pct as well

Reply:
Fixed.

Reply:
INSERT INTO pay (salary, commission_pct, cardNr)
SELECT salary, commission_pct, cardNr
FROM employees;

Reply:
Fixed.